src/components/Navbar.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] As required by MIcroverse, you are not meant to use any LInters diable rule in your codebase. The Linters are there to make sure you follow the best practices. I'll suggest you remove these and try to fix the errors. :smile: :small_airplane: :+1:

Reply:
The other reviewer tells me I can use it because when I solve it ask me to put it in the other dependency list. When I put it in dev-dependencies ask me to put it in dependency, and vice versa.